shared/login/recover-password/explain-device/index.tsx

```diff
@@ -41,7 +41,7 @@ const ExplainDevice = (props: Props) => {
             ) : (
               <Kb.Text type="Body">Settings</Kb.Text>
             )}
-            <Kb.Text type="Body">> Your account, and change your</Kb.Text>
```

Comment:
incorrect!

Reply:
Shouldn't there be a &gt; though? "Settings > Your account"

Reply:
Oh alrite I'll put it back

Reply:
@cjb done